caikit/runtime/grpc_server.py

```diff
@@ -103,6 +114,31 @@ def __init__(
                 self.inference_service.service, self.server
             )
 
+        if self.enable_inference_jobs:
+            # Initialize the job management servicer and create a function wrapper
+            self._prediction_job_management_servicer = (
+                PredictionJobManagementServicerImpl()
+            )
+            self.server = CaikitRuntimeServerMultiFuncWrapper(
```

Comment:
I haven't made it far enough yet, but I'm not understanding the need for this `MultiFuncWrapper` stuff yet...